doc/nrf/test_and_optimize/optimizing/power_general.rst

```diff
@@ -86,3 +91,103 @@ For example, for the nRF9160 SiP, see the `Electrical specification of nRF9160`_
+
+To enable suspending peripherals when CPU goes to sleep, set the :kconfig:option:`CONFIG_PM_DEVICE` Kconfig option to ``y``.
+
+.. enable_device_pm_end
+
+Put the external flash into sleep mode in inactivity periods
+============================================================
+
+When the CPU goes to sleep, some of the peripherals are suspended by their drivers.
+However, the driver is not always able to know the application behavior and optimally handle the peripheral state.
+
+One of such cases is the external flash usage by the applications.
+It is typically used very rarely and only for the Device Firmware Upgrade purposes.
+For this reason, you might want to suspend the external flash for majority of time and have it resumed to the active state only if needed.
+The Device Firmware Upgrade case is properly handled in the |NCS|, but for other proprietary use cases, you should handle state changes in your own implementation.
+
+For example, to control the QSPI NOR external flash, you can use the following implementation:
+
+.. code-block:: C++
+
+    #include <zephyr/pm/device.h>
+
+    const auto * qspi_dev = DEVICE_DT_GET(DT_INST(0, nordic_qspi_nor));
+    if (device_is_ready(qspi_dev))
+    {
+        // Put the peripheral into suspended state.
+        pm_device_action_run(qspi_dev, PM_DEVICE_ACTION_SUSPEND);
+
+        // Resume the peripheral from the suspended state.
+        pm_device_action_run(qspi_dev, PM_DEVICE_ACTION_RESUME);
+    }
+
+Configure radio transmitter power
+=================================
+
+.. radio_power_start
+
+The radio transmitter power (radio TX power) has a significant impact on the device power consumption.
+The higher the transmitting power, the greater the wireless communication range, which leads to higher power consumption.
+Make sure to choose the optimal configuration for your specific use case.
+
+.. radio_power_end
+
+.. _app_power_opt_protocols:
+
+Protocol-specific recommendations
+*********************************
+
+Besides applying `General recommendations`_, read the following subsections for more information on how to optimize specific subsystems.
+
+Bluetooth mesh
+==============
+
+The Bluetooth mesh protocol offers the :ref:`ug_bt_mesh_configuring_lpn` feature for optimizing the power consumption of the Bluetooth mesh devices.
+
+Matter
+======
+
+To optimize the power consumption of your Matter application, complete the actions listed on the :ref:`ug_matter_device_low_power_configuration` page.
+
+Thread
+======
+
+The Thread protocol offers the :ref:`Sleepy End Device types <thread_sed_ssed>` feature for optimizing the power consumption of the Thread devices.
+
+Wi-Fi
+=====
+
+The Wi-Fi protocol introduces the power save mechanism that allows the Station device (STA) to spend the majority of time in a sleep state and wake-up periodically to check for pending traffic.
```

Comment:
```suggestion
The Wi-Fi protocol introduces the power save mechanism that allows the Station (STA) device to spend the majority of time in a sleep state and wake up periodically to check for pending traffic.
```

Reply:
> 

@sachinthegreen @bama-nordic : Can you please take a look at the Wi-Fi section here to verify the info and see if anything else is required here?